bftengine/src/bftengine/messages/ClientRequestMsg.hpp

```diff
@@ -75,14 +76,20 @@ class ClientRequestMsg : public MessageBase {
 
   void validateImp(const ReplicasInfo& repInfo) const;
 
+  // Returns a pair of pointer and size to the extra buffer which was allocated during initialisation
+  std::pair<char*, uint32_t> getExtraBufPtr() {
```

Comment:
nit: If we need only length then also this will result into buffer pointer calculation.

Reply:
True, but we usually need both. Even if the pointer is not used I think it's a small price to pay.